src/main/java/Dog.java

```diff
@@ -15,17 +16,18 @@ public Dog(String name, String owner){
         System.out.println("It's Owner is "+name);
     }
 
-    public void run(){
+    public void dorun(){
         System.out.println(name+" is runing");
         this.run=true;
     }
 
-    public  void stop(){
+    public  void dostop(){
         System.out.println(name+" stopped");
         this.run=false;
+        this.walk=false;
     }
 
-    public void eat(){
+    public void doeat(){
         if(this.run==true){
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the literal "true" boolean value. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1125)